code/datums/keybindings/emote.dm

```diff
@@ -0,0 +1,594 @@
+/datum/keybinding/emote
+	category = KB_CATEGORY_EMOTE_GENERIC
+	var/datum/emote/linked_emote
+
+/datum/keybinding/emote/can_use(client/C, mob/M)
+	return isliving(M) && (M.stat != DEAD) && ..() //piss off, ghost
```

Comment:
Hmm.

So ghosts being able to use some emotes is a feature, not a bug, that was brought in during the emote refactor. Ghost stat is DEAD, so this would also silently prevent them from using emotes.
There's a define in `say.dm` that controls the emotes that ghosts are able to do:
```
#define USABLE_DEAD_EMOTES list("*flip", "*spin")
```
It might be worth moving that into `code/__DEFINES/emotes.dm` and referencing it here, I just left it in `say.dm` since it was only used there.

You also shouldn't necessarily need to check `isliving` since the emote datums should be able to handle that for you, and that'd probably also prevent ghosts from using emotes.